### DISCUSSION TITLE
Error in submitting to Codeforces

### DISCUSSION THREAD

**Discussion by Ju5T1N-VN** [cpeditor/cpeditor#1374] [Q&A] [answered]
I don't know why but when I used CP Editor to submit my code to Codeforces, the error message appeared like this:
![image](https://github.com/cpeditor/cpeditor/assets/172887547/2532fbb2-e70d-4123-80f9-bad1433d1a5f)

I have also tried to submit my code to a problem in Codeforces Round 952 (Div. 4) and it still worked:
![image](https://github.com/cpeditor/cpeditor/assets/172887547/4f5ef0fb-475c-4df0-8277-c11df2488bd6)

I'm currently using CP Editor 6.11.2 and CF Tool 1.0.0

**Comment by ouuan** [comment, marked as answer]
`gym` URLs are currently not supported
